### PR TITLE
Add metadata for google to have a better description and fixed downloadsURL Mixin

### PR DESCRIPTION
**Pull request by MartinMuzatko**
This is not a very convenient google description:

image

Also, I switched to native fetch (and included the polyfills required to render on IE8) in order to use `Promise.all`.

As discussed on Riot:

> About the Download problem: this.$http.get does not return a real Promise, but a PromiseObj, probably wrapped by the $http library (VueResource I assume)
> image
> After looking into their documentation and source code, I can tell that vue-resource is garbage.. They don't even document that you have to use this.$promise.all instead of Promise.all. You only find numerous issues without a documentation D:

### CODE REVIEW

**Comment by Croydon** [comment]
> Also, technically speaking this is not including third party content (otherwise we could also say jQuery or Vue is third party content)

The keyword was external - from third-party servers, directly included.

**Comment by Fohlen** [comment]
@MartinMuzatko unfortunately you didn't push a branch in this repo but forked it, which is why I can hardly alter your changes. Can you please use https://www.npmjs.com/package/polyfills instead of including it via a CDN? I know bli bla blub but we had this discussion a while ago and I'm sure @Croydon didn't change his mind about this.

**Comment by MartinMuzatko** [comment]
@Fohlen I wanted to create a branch, but I was not allowed to, I need to be in the site team.

For async/await: We are already using babel to transpile code.
And async await is just syntax sugar for Promises :)

https://caniuse.com/#feat=async-functions

Btw, async is already supported by all evergreen browsers.

File: index.html [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
-  <meta name="twitter:image" content="https://inexor.org/src/assets/logo/inexor_cube_beta.png">
+  <meta name="twitter:image" content="https://inexor.org/src/assets/logo/inexor_cube_beta.png" />
   
   <!-- Open Graph data -->
   <meta property="og:title" content="Inexor" />
   <meta property="og:type" content="website" />
   <meta property="og:url" content="https://inexor.org/" />
   <meta property="og:image" content="https://inexor.org/src/assets/logo/inexor_cube_beta.png" />
-  <meta property="og:description" content="Inexor will end up being kind of a sandbox, and in fact it is already. Users should be able to create their own themes and
+  <meta property="og:description" content="Inexor will end up being kind of a sandbox, and in fact it is already. Users should be able to create their own themes an /d
```
Comment (Croydon) [inline index.html]: False search and replace

Comment (MartinMuzatko) [reply index.html]: doh! fixing it :)

**Comment by MartinMuzatko** [comment]
not valid anymore. :)

**Pull request closed without merging**